sdk/keyvault/azure-security-keyvault-administration/CHANGELOG.md

```diff
@@ -1,16 +1,6 @@
 # Release History
 
-## 4.0.0-beta.2 (Unreleased)
-
-### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
-
-### Other Changes
-
-## 4.0.0-beta.1 (2023-02-07)
+## 4.0.0-beta.1 (Unreleased)
```

Comment:
Either way is fine, but if we make the changes in the future, we'll rather have to release as beta.2 anyways (the git tag for `beta.1` is out already). So I think the following is the better representation of the current state of things. During the re-listing of the release, we'll update the release date for `beta.1` once again.

```suggestion
## 4.0.0-beta.2 (Unreleased)

### Features Added

### Breaking Changes

### Bugs Fixed

### Other Changes

## 4.0.0-beta.1 (Unreleased)
```